lib/fluent/plugin/github-activities/users_manager.rb

```diff
@@ -23,80 +23,82 @@
 require "fluent/plugin/github-activities/safe_file_writer"
 
 module Fluent
-  module GithubActivities
-    class UsersManager
-      DEFAULT_LAST_EVENT_TIMESTAMP = -1
+  module Plugin
+    module GithubActivities
+      class UsersManager
+        DEFAULT_LAST_EVENT_TIMESTAMP = -1
 
-      def initialize(params={})
-        @users = params[:users]
+        def initialize(params={})
+          @users = params[:users]
 
-        @positions = {}
-        @pos_file = params[:pos_file]
-        @pos_file = Pathname(@pos_file) if @pos_file
-      end
+          @positions = {}
+          @pos_file = params[:pos_file]
```

Comment:
Is there any chance to use storage plugin here like as https://github.com/reevoo/fluent-plugin-systemd/pull/31?